### ISSUE TITLE
Getting changed text from textctrl update event

### ISSUE THREAD

**Issue by padmadevd** [wxWidgets/wxWidgets#23605]
Pls add a feature like getting the text inserted or removed from text update event of textctrl. Now I only get the full text in the control. But I only need the changed text. I can get it by comparing the new text from previous stored text. But I think it is inefficient

**Comment by vadz** [comment]
Yes, it would be useful to have the text being inserted, but I'm not sure how easy/difficult is it to implement this (for the text being removed you can simply look at the old selection).

Any help/PRs adding this would be welcome.
